tests/test_data/integration.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am not very familiar with test outputs, but I think xunit is popular.

It may be worth looking to see if we can make qt generate xunit format, e.g.:

https://doc.qt.io/qt-5/qtest-overview.html#options

Using this, it may be possible to reduce our two formats down to just one.

Reply:
If we can do that, then perhaps the format should be change from `gtest` to `xunit` to indicate that it is more generic.  (I would definitely check the xunit format---perhaps neither of them are actually "standard" and have incompatible differences).

Reply:
@orwonthe did you see this comment?

Reply:
I added explicit xunit format. This is tested with qttest output with `--xunitxml` format flag.